### PR TITLE
bpo-34344 Fixed AbstractEventLoopPolicy.get_event_loop docstrings

### PR DESCRIPTION
**Pull request by idomic**
https://bugs.python.org/issue34344

### CODE REVIEW

File: Lib/asyncio/events.py [hunk 1]
```diff
@@ -577,13 +577,10 @@ class AbstractEventLoopPolicy:
...
 
     def get_event_loop(self):
-        """Get the event loop for the current context.
 
-        Returns an event loop object implementing the BaseEventLoop interface,
-        or raises an exception in case no event loop has been set for the
-        current context and the current policy does not specify to create one.
-
-        It should never return None."""
+        """ Get the event loop for the current context.
```
Comment (brandtbucher) [inline Lib/asyncio/events.py]: No leading space here:

```suggestion
        """Get the event loop for the current context.
```

File: Lib/asyncio/events.py [hunk 2]
```diff
@@ -577,13 +577,10 @@ class AbstractEventLoopPolicy:
...
+            Returns an instance of EventLoop or raises an exception.
```
Comment (brandtbucher) [inline Lib/asyncio/events.py]: No hanging indent here:

```suggestion
         Returns an instance of EventLoop or raises an exception.
```

File: Lib/asyncio/events.py [hunk 3]
```diff
@@ -577,13 +577,10 @@ class AbstractEventLoopPolicy:
...
+       """
```
Comment (brandtbucher) [inline Lib/asyncio/events.py]: Bad indent here:

```suggestion
        """
```

**Review by brandtbucher — CHANGES_REQUESTED** [review]
Thanks @idomic, and welcome to CPython!

This change looks good, but I think it was made in the wrong place. This new docstring should be moved to the `get_event_loop` method on **line 632** of this same file. The current docstring for line 580 is perfectly fine as-is.

Besides that, I've got a couple of formatting fixes that should be incorporated. See below:

**Comment by idomic** [comment]
Thanks @brandtbucher for the warm welcome and the detailed comments.
I've removed the past change and fixed the remarks on the docstring at line 632.

File: Lib/asyncio/events.py [hunk 4]
```diff
@@ -630,9 +630,8 @@ def __init__(self):
         self._local = self._Local()
 
     def get_event_loop(self):
-        """Get the event loop.
-
-        This may be None or an instance of EventLoop.
+        """Get the event loop for the current context.
+           Returns an instance of EventLoop or raises an exception.
```
Comment (brandtbucher) [inline Lib/asyncio/events.py]: Looks good @idomic! One more tiny formatting fix (hanging indent):

```suggestion
        Returns an instance of EventLoop or raises an exception.
```

**Review by brandtbucher — APPROVED** [review]

File: Lib/asyncio/events.py [hunk 5]
```diff
@@ -630,9 +630,8 @@ def __init__(self):
         self._local = self._Local()
 
     def get_event_loop(self):
-        """Get the event loop.
-
-        This may be None or an instance of EventLoop.
+        """Get the event loop for the current context.
+        Returns an instance of EventLoop or raises an exception.
```
Comment (1st1) [inline Lib/asyncio/events.py:635]: Need a newline before "Returns".  See https://www.python.org/dev/peps/pep-0257/ for the reference.

**Review by 1st1 — APPROVED** [review]

**Comment by brandtbucher** [comment]
Congrats on your first PR @idomic!

**Pull request merged**